server/routes/index.js

```diff
@@ -0,0 +1,13 @@
+const route = require("express").Router();
+const userR = require("../routes/userRoutes");
+const articleR = require("./articleRoute");
+
+route.use("/users", userR);
+route.use("/articles", articleR);
+
+route.get("/*", (req, res) => {
```

Comment:
tips: kamu bisa menggunakan ini untuk check 404 page not found, jadi disesuaikan penggunaannya